nipype/interfaces/mrtrix3/connectivity.py

```diff
@@ -228,3 +228,91 @@ def _list_outputs(self):
         outputs = self.output_spec().get()
         outputs['out_file'] = op.abspath(self.inputs.out_file)
         return outputs
+
+
+class LabelConvertInputSpec(CommandLineInputSpec):
+    in_file = File(
+        exists=True,
+        argstr='%s',
+        mandatory=True,
+        position=-4,
+        desc='input anatomical image')
+    in_lut = File(
+        exists=True,
+        argstr='%s',
+        mandatory=True,
+        position=-3,
+        desc='get information from '
+        'a basic lookup table consisting of index / name pairs')
+    in_config = File(
+        exists=True,
+        argstr='%s',
+        position=-2,
+        desc='connectome configuration file')
+    out_file = File(
+        'parcellation.mif',
+        argstr='%s',
+        mandatory=True,
+        position=-1,
+        usedefault=True,
+        desc='output file after processing')
+    spine = File(
+        argstr='-spine %s',
+        desc='provide a manually-defined '
+        'segmentation of the base of the spine where the streamlines'
+        ' terminate, so that this can become a node in the connection'
+        ' matrix.')
+    nthreads = traits.Int(
+        argstr='-nthreads %d',
+        desc='number of threads. if zero, the number'
+        ' of available cpus will be used',
+        nohash=True)
+
+
+class LabelConvertOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='the output response file')
+
+
+class LabelConvert(MRTrix3Base):
+    """
+    Re-configure parcellation to be incrementally defined.
+
+    Example
+    -------
+
+    >>> import nipype.interfaces.mrtrix3 as mrt
+    >>> labels = mrt.LabelConvert()
+    >>> labels.inputs.in_file = 'aparc+aseg.nii'
+    >>> labels.inputs.in_config = 'mrtrix3_labelconfig.txt'
+    >>> labels.inputs.in_lut = 'FreeSurferColorLUT.txt'
+    >>> labels.cmdline                               # doctest: +ELLIPSIS
+    'labelconvert aparc+aseg.nii FreeSurferColorLUT.txt mrtrix3_labelconfig.txt parcellation.mif'
+    >>> labels.run()                                 # doctest: +SKIP
+    """
+
+    _cmd = 'labelconvert'
+    input_spec = LabelConvertInputSpec
+    output_spec = LabelConvertOutputSpec
+
+    def _parse_inputs(self, skip=None):
+        if skip is None:
+            skip = []
+
+        if not isdefined(self.inputs.in_config):
+            from distutils.spawn import find_executable
+            path = find_executable(self._cmd)
```

Comment:
We're looking at dropping the `distutils` dependency (#2348). Can you use `nipype.utils.filemanip.which` instead?